bigbluebutton-html5/imports/ui/components/screenshare/service.js

```diff
@@ -58,14 +59,29 @@ const shareScreen = (onFail) => {
   }
 
   BridgeService.getScreenStream().then((stream) => {
-    KurentoBridge.kurentoShareScreen(onFail, stream);
+    window.screenShareStream = stream;
+    //splitting up the video and audio stream into separate stream objects
+    // because Kurento does not like getting both audio and video data.
+    let videoStream = new MediaStream(stream.getVideoTracks())
+    KurentoBridge.kurentoShareScreen(onFail, videoStream)
+    if(typeof stream.getAudioTracks == "function" && stream.getAudioTracks().length > 0){
+      let audioStream = new MediaStream(stream.getAudioTracks())
+      AudioManager.joinScreenShareStream(audioStream)
```

Comment:
@nanovim have you tested this triggering a microphone reconnect? Don't think it'll pick up the audio mixer again if that happens.

Reply:
I actually did not think of that. 
Maybe it does, but probably not. 
I guess this needs a fix.